releasenotes/notes/rename-xccdf-configuration-flag-19129eabe11a7bcc.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    The `compliance_config.xccdf.enabled` configuration has been renamed to `compliance_config.host_benchmarks.enabled`.
```

Comment:
What about having only the `deprecations` section ?

Reply:
I don't know. Some change do only `deprecations`, some other do both. Doing both might be useful since the change appear in different sections.

Reply:
Thanks. I've removed the `features` section.